src/http.ts

```diff
@@ -258,6 +284,12 @@ export class HttpClient {
         throw new InvalidUPCIdentifier(`${info.action} request failed`)
       }
 
+      // GetReport returns a string that should be treated as a file instead of XML data
+      // http://docs.developer.amazonservices.com/en_CA/reports/Reports_GetReport.html
+      if (info.action === 'GetReport') {
```

Comment:
Yep, was thinking the same but I thought this might've been fine since this was the only API, so far, that had that special case I'll create an issue and see if I encounter anything like it